modules/move-etcd-different-disk.adoc

```diff
@@ -10,20 +10,52 @@ You can move etcd from a shared disk to a separate disk to prevent or resolve pe
 
 The Machine Config Operator (MCO) is responsible for mounting a secondary disk for {product-title} {product-version} container storage.
 
+[NOTE]
+====
+This procedure supports NVME devices and devices where the name starts with `/dev/sd*` or `/dev/vd*`. For an NVME device, use `/dev/nvme*[0-9]\*n*` as the device name.
+====
+
+This script is for `/etc/find-secondary-device`. Replace the device name according to your environment. Decode and replace the device name in the machine configuration file.
```

Comment:
Including a script as part of the introductory information feels strange to me. If the script described here is used in the procedure, should creating it be the first step?

Reply:
We don't need to create the script, it is already part of the machine-config but it is encoded. In order for the users to be aware of the script we have shown it. Also the script needs changes as per customer environment. If the user has disks which starts with /dev/sd* or /dev/vd* or for NVME, they need to change the device name in the script accordingly.

Reply:
If the script could need changes, I think it will be better to explain what needs to be changed and how to encode it before generating the Machine Config to avoid confusion. I understand it should be before the current step 2, right?